website/settings/defaults.py

```diff
@@ -189,9 +189,12 @@ def parent_dir(path):
 NO_LOGIN_WAIT_TIME = timedelta(weeks=4)
 WELCOME_OSF4M_WAIT_TIME = timedelta(weeks=2)
 NO_LOGIN_OSF4M_WAIT_TIME = timedelta(weeks=6)
-NEW_PUBLIC_PROJECT_WAIT_TIME = timedelta(hours=24)
```

Comment:
[Soft Blocker] Curious why remove simply remove this `NEW_PUBLIC_PROJECT_WAIT_TIME` while both four are still in `queue_mail_types`?

```python
queue_mail_types = {
    NO_ADDON_TYPE: NO_ADDON,
    NO_LOGIN_TYPE: NO_LOGIN,
    NEW_PUBLIC_PROJECT_TYPE: NEW_PUBLIC_PROJECT,
    WELCOME_OSF4M_TYPE: WELCOME_OSF4M
}
```